.github/workflows/ci.yml

```
@@ -27,6 +27,16 @@ jobs:
 
   dashboard-api-tests:
     runs-on: ubuntu-latest
+     services:
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)